packages/bot/src/run-bot.ts

```diff
@@ -0,0 +1,91 @@
+/*import AgonesSDK from '@google-cloud/agones-sdk'
```

Comment:
Getting this error when check-errors runs:

src/run-bot.ts(1,1): error TS1208: 'run-bot.ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.

I see that the `dev` npm script calls this. Should this just be deleted, along with the script?

Reply:
Removed